### PR TITLE
📊 Dynabench - AI performance on various benchmarks

### PR DESCRIPTION
**Pull request by veronikasamborska1994**

### CODE REVIEW

File: etl/steps/data/garden/artificial_intelligence/2024-04-02/dynabench.meta.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+dataset:
+  update_period_days: 365
+
+tables:
+  dynabench:
+    variables:
+      performance:
+        title: Performance
+        description_short: Test scores of AI relative to human performance. Human performance, as the benchmark, is set to zero. The capability of each AI system is normalized to an initial performance of - 1.
```
Comment (spoonerf) [inline etl/steps/data/garden/artificial_intelligence/2024-04-02/dynabench.meta.yml:36]: Tiny nit but threw me slightly when looking at the default subtitle, I'd remove the space in '- 1'.

File: etl/steps/data/garden/artificial_intelligence/2024-04-02/dynabench.meta.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+dataset:
+  update_period_days: 365
+
+tables:
+  dynabench:
+    variables:
+      performance:
+        title: Performance
```
Comment (spoonerf) [inline etl/steps/data/garden/artificial_intelligence/2024-04-02/dynabench.meta.yml:35]: Maybe 'AI test scores relative to human benchmark', or something would be a more informative title.

File: snapshots/artificial_intelligence/2024-04-02/dynabench.xlsx.dvc [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+meta:
+  origin:
+    # Data product / Snapshot
+    title: "Dynabench: Rethinking Benchmarking in NLP"
+    description: |-
+      This dataset captures the progression of AI evaluation benchmarks, reflecting their adaptation to the rapid advancements in AI technology. The benchmarks cover a wide range of tasks, from language understanding to image processing, and are designed to test AI models' capabilities in various domains. The dataset includes performance metrics for each benchmark, providing insights into AI models' proficiency in different areas of machine learning research.
+
+      - MMLU (Massive Multitask Language Understanding): Tests AI's understanding across multiple disciplines, with metrics for knowledge comprehension and application.
```
Comment (spoonerf) [inline snapshots/artificial_intelligence/2024-04-02/dynabench.xlsx.dvc:10]: This hasn't formatted very nicely in my local set up. Does it look okay for you? 
Screenshot 2024-04-02 at 15 15 19

**Review by spoonerf — APPROVED** [review]
Looks good just a few nits!

File: etl/steps/data/garden/artificial_intelligence/2024-04-02/dynabench.meta.yml [hunk 4]
```diff
@@ -0,0 +1,40 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    processing_level: minor
+    description_processing: >
+      We mapped the benchmarks to their respective domains based on a review of each benchmark's primary focus and the specific capabilities it tests within AI systems:
```
Comment (spoonerf) [inline etl/steps/data/garden/artificial_intelligence/2024-04-02/dynabench.meta.yml:6]: Oh sorry it's this bit that hasn't formatted nicely

Comment (spoonerf) [reply etl/steps/data/garden/artificial_intelligence/2024-04-02/dynabench.meta.yml:6]: Maybe you need a |- instead of >

Comment (spoonerf) [reply etl/steps/data/garden/artificial_intelligence/2024-04-02/dynabench.meta.yml:6]: Yeh you just need `description_processing: |-` instead

Comment (veronikasamborska1994) [reply etl/steps/data/garden/artificial_intelligence/2024-04-02/dynabench.meta.yml:6]: ah yeaa sorry was in a bit of a rush with this one and missed the formatting :) thanks a lot for spotting it though!

**Pull request merged**